components/BaseRadioButton/index.js

```diff
@@ -0,0 +1,25 @@
+import * as Styled from './styles';
+import { useRadioContext } from './radio-context';
+
+function BaseRadioButton({ children, value, ...inputProps }) {
+  const [state, onChange] = useRadioContext();
+  const checked = value === state;
+
+  return (
+    <Styled.Radio>
```

Comment:
Un overwrite de estilos no debería venir desde aquí?
```suggestion
    <Styled.Radio {...inputProps} >
```

Reply:
Me puse en el caso que quisiéramos manejar el radio button standalone, y facil le pasen propiedades como `onChange`, `checked`,  etc